drivers/counter/counter_nrfx_rtc.c

```diff
@@ -333,7 +328,7 @@ static const struct counter_driver_api counter_nrfx_driver_api = {
 			    DT_NORDIC_NRF_RTC_RTC_##idx##_IRQ_PRIORITY,	       \
 			    nrfx_isr, nrfx_rtc_##idx##_irq_handler, 0);	       \
 		const nrfx_rtc_config_t config = {			       \
-			.prescaler = CONFIG_COUNTER_RTC##idx##_PRESCALER,      \
+			.prescaler = DT_NORDIC_NRF_RTC_##idx##_PRESCALER,      \
```

Comment:
You need to use the macros generated for aliases, i.e. `DT_NORDIC_NRF_RTC_RTC_x_*` in this case, since the indexes in the standard macros generated for all nodes with a given `compatible` (i.e. `DT_NORDIC_NRF_RTC_x_*` here)  may not at all correspond to the numbering of instances (for instance, `DT_NORDIC_NRF_RTC_0_*` may actually refer to RTC2); they are "python dependent" and only enabled nodes are counted.

Reply:
done